tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -2372,4 +2371,77 @@ public String format(
                 nf3.locale(uLocale).format(outputAmount.doubleValue()).toString();
         return com.ibm.icu.text.MessageFormat.format(pattern, cldrFormattedNumber);
     }
+
+    public enum QuantityGroup {
+        Duration("duration", "year-duration"),
+        Length("length"),
```

Comment:
Here again a brief comment explaining the significance of the ordering could be helpful

Reply:
I also have a proposal for breaking apart the Units Other (which also ends up breaking apart a couple of others. It's just draft for now until we can discuss.